apps/main/src/App.tsx

```diff
@@ -31,6 +31,7 @@ const App: React.FC = () => {
         <Route path="assets-products/*" element={<AssetsProductsPage />} />
         <Route path="channels-and-services/*" element={<ChannelsAndServicesPage />} />
         <Route path="history-interactions/*" element={<HistoryInteractionsPage />} />
+        <Route path="records/*" element={<HistoryInteractionsPage />} />
```

Comment:
Esqueceste-te de mudar aqui o nome do componente

```suggestion
        <Route path="records/*" element={<RecordsPage />} />
```